src/main/java/nl/quintor/studybits/indy/wrapper/util/PoolUtils.java

```diff
@@ -45,6 +45,24 @@ private static File createGenesisTxnFile(String filename, String testPoolIp) thr
 		return file;
 	}
 
+	private static File createGenesisTxnFile(String filename, String[] txns) throws IOException {
```

Comment:
I would refactor the other function to make use of this one, to avoid duplicate code